obs-studio-server/source/nodeobs_api.cpp

```diff
@@ -427,14 +428,16 @@ void OBS_API::OBS_API_initAPI(
 	}
 	pathConfigDirectory = args[0].value_str.c_str();
 	appdata_path        = args[0].value_str.c_str();
-	appdata_path += "/node-obs/";
 
 	/* libobs will use three methods of finding data files:
 	* 1. ${CWD}/data/libobs <- This doesn't work for us
 	* 2. ${OBS_DATA_PATH}/libobs <- This works but is inflexible
 	* 3. getenv(OBS_DATA_PATH) + /libobs <- Can be set anywhere
 	*    on the cli, in the frontend, or the backend. */
 	obs_add_data_path((g_moduleDirectory + "/libobs/data/libobs/").c_str());
+	slobs_plugin = appdata_path.substr(0, appdata_path.size() - 14);
```

Comment:
What is 14 here?

Reply:
I could probably make it easier to read. The frontend is giving us the full appdata path of the app, I need to go one step up to only get the appdata path.

Reply:
I see. I can see how this is a bit problematic. I'm not sure how I feel on us relying on the frontend for this entirely. Ideally, we should probably use SHGetKnownFolder and FOLDERID_RoamingAppData. To clarify intent, this should probably be something like `strlen("/slobs-client") + 1` instead of 14.

Reply:
Actually, re-reading this, I'm not sure it does what it intended. I'm going to look at this in a debugger.

Reply:
Oh, Apparently, the above is old and you changed this already to 13. It should be `strlen("/slobs-client")` instead of 13.

Reply:
Yeah I can change that to clarify the intent. I would prefer to rely on what the frontend is passing us. If the frontend relies on a wrong appData, the backend should use what the frontend uses otherwise it will cause a lot of issues.